doxygen_test/PolygonBuildData[] polygons
